src/indexer/services/monitoringService/index.ts

```diff
@@ -0,0 +1,42 @@
+import { CronJob } from 'cron'
+import TransferRepository from '../../repository/transfer';
+import { logger } from '../../../utils/logger';
+import { DateTime } from 'luxon';
+import { NotificationSender } from './notificationSender';
+
+export async function checkTransferStatus(transferRepository: TransferRepository): Promise<void>{
+
+    const notificationSender = new NotificationSender("region")
+
+    const transfers = await transferRepository.findTransfersByStatus("pending")
+    console.log("--- Cronjob is running ---")
+
+    for (let transfer of transfers){
+        if (transfer.timestamp) {
+            let duration = (new Date()).getTime() - transfer.timestamp.getTime()
+            const durationInMins = duration / 1000 / 60 
```

Comment:
Can you write some util function (or maybe it already exists in the standard library)?
Always best to create a util function test it and name it properly and the code is ultimately more readable and correct.